test/node/double.test.ts

```diff
@@ -225,6 +225,59 @@ describe('BSON Double Precision', function () {
         });
       });
     });
+
+    describe('fromString', () => {
+      const acceptedInputs = [
+        ['zero', '0', 0],
+        ['non-leading zeros', '45000000', 45000000],
+        ['zero with leading zeros', '000000.0000', 0],
+        ['positive leading zeros', '000000867.1', 867.1],
+        ['negative leading zeros', '-00007.980', -7.98],
+        ['positive integer with decimal', '2.0', 2],
+        ['zero with decimal', '0.0', 0.0],
+        ['Infinity', 'Infinity', Infinity],
+        ['-Infinity', '-Infinity', -Infinity],
+        ['NaN', 'NaN', NaN],
+        ['basic floating point', '-4.556000', -4.556],
+        ['negative zero', '-0', -0]
+      ];
+
+      const errorInputs = [
+        ['commas', '34,450'],
+        ['exponentiation notation', '1.34e16'],
+        ['octal', '0o1'],
+        ['binary', '0b1'],
+        ['hex', '0x1'],
+        ['empty string', ''],
+        ['leading and trailing whitespace', '    89   ', 89],
+        ['fake positive infinity', '2e308'],
+        ['fake negative infinity', '-2e308'],
+        ['fraction', '3/4'],
+        ['foo', 'foo']
+      ];
+
+      for (const [testName, value, expectedDouble] of acceptedInputs) {
+        context(`when case is ${testName}`, () => {
+          it(`should return Double that matches expected value`, () => {
```

Comment:
Same comment here about test titles.

Reply:
Just changed them, going to push after my tests run!